.gitmodules

```diff
@@ -31,6 +31,10 @@
 	path = amazon-s3-plugins
 	url = ../../data-integrations/amazon-s3-plugins
 	branch = release/1.12
+[submodule "azure"]
+	path = azure
+	url = ../../data-integrations/azure
+	branch = release/1.7
```

Comment:
@shifuxu0301 can you change this to release/1.8 and update the submodule hash?

Reply:
My ongoing PR is not merged into release/1.8 yet. And also the version is also not bumped yet. In this particular case, is it okay for me to upgrade to 1.8?

Reply:
yeah, that's ok. We want the azure shipped with 6.2.0 to be version 1.8

Reply:
Got it. Will do.